File: Documentation/API/Utilities/IsOnTheBeat.cs
using System;
using RhythmGameUtilities;

const int bpm = 120;
const float currentTime = 10f;
const float delta = 0.05f;

var isOnTheBeat = Utilities.IsOnTheBeat(bpm, currentTime, delta);

Console.WriteLine(isOnTheBeat ? "Is on the beat!" : "Is not on the beat!"); // "Is on the beat!"

File: Documentation/API/Utilities/IsOnTheBeat.js
import RhythmGameUtilities from '@rhythm-game-utilities/core';

const bpm = 120;
const currentTime = 10;
const delta = 0.05;

const isOnTheBeat = RhythmGameUtilities.IsOnTheBeat(bpm, currentTime, delta);

console.log(isOnTheBeat ? 'Is on the beat!' : 'Is not on the beat!'); // Is on the beat!
